org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/ExecuteClientCommandTest.java

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
```

Comment:
Pivotal

Reply:
Ah... yeah. The headers were all added automagically by Eclipse (you must have configured it that way in the projects). And I didn't pay attention so I didn't notice. 

I'll make pass through all the copyright headers of all the changed files.